tests/unit/introspection.tcl

```diff
@@ -201,6 +201,9 @@ start_server {tags {"introspection"}} {
             cluster-port
             oom-score-adj
             oom-score-adj-values
+            protected-configs
+            debug-commands-disabled
+            module-commands-disabled
```

Comment:
unrelated to shit actual block.

you have to enable these in the `tests/assets/default.conf` file, otherwise the tests will not run (now that i've changed the default value, which was the whole point of this PR. i.e. more secure defaults)